client/src/components/LoggedInLayout.tsx

```diff
@@ -32,16 +33,21 @@ const LoggedInLayout = () => {
         <Sidebar />
         <Main>
           <Header />
-          <Suspense fallback={<StandardLoadingState />}>
-            <div
-              className={cx({
-                'opacity-30 transition-opacity duration-100':
-                  navigation.state === 'loading',
-              })}
-            >
-              <Outlet />
+          <div className="grid grid-cols-[1fr,minmax(250px,_300px)]">
```

Comment:
this is just an prototype :) 

If we do this we could think about refactoring some parts of the layout :)

for example in the Spotify App the user profile is now in the center part of the layout (and for us is fixed on the top right)